[DOC-13153]: Feedback on Query Service Metrics | Couchbase Docs (#3818)
couchbase/docs-server: modules/metrics-reference/pages/query-service-metrics.adoc

The following Query Service metrics can be queried by means of the REST APIs described in xref:rest-api:rest-statistics.adoc[Statistics].

> As a brief introduction however, we will first look at how you would go about building a REST command for the `n1ql_active_requests` metric.

Reviewer: Style rules say to avoid first person "we":

```suggestion
As a brief introduction however, this is how you would go about building a REST command for the `n1ql_active_requests` metric.
```
